Makefile

```diff
@@ -3,11 +3,19 @@ TIMESTAMP ?= $(shell date +%s)
 # these can vary by 1 second
 export TIMESTAMP
 
+ifeq ($(PRIVATE),true)
+  REPOSITORY_SUFFIX:=-private
+endif
+
+SERVICES_SEPARATOR=,
+BUILD_SERVICES?=auth,blob,data,migrations,prescription,task,tools
+BUILD_SERVICES:=$(subst $(SERVICES_SEPARATOR), ,$(BUILD_SERVICES))
+
 ROOT_DIRECTORY:=$(realpath $(dir $(realpath $(lastword $(MAKEFILE_LIST)))))
```

Comment:
Is this not just $(CURDIR), which is automatically set by make.

Oh, this isn't a modified line, oh well, ignore.